lib/constants/whitelabel-providers.ts

```diff
@@ -33,6 +33,36 @@ const WHITELABEL_PROVIDERS: WhitelabelProvider[] = [
       { label: 'Terms of Service', href: '/tos', icon: ProfileMenuIcons.FileText },
     ],
   },
+  {
+    slug: 'opensource',
```

Comment:
We can not have this on E2E/CI since there are some tests that use opensource entities. We'll need to push this provider inside a conditional.